### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by wangzeyu135798** [snap-stanford/pretrain-gnns#26]
Hi:
    After reading your paper and code. I have two questions.
    First. In pretrain-gnns-master/model_gin, there are several supervised_.pth files, how to get them? 
    Second, in README.md Fine-tuning, there is a parameter dataset, which is downstream_dataset. While in finetune.py and finetune_tune.sh , there is no this dataset parameter. Why?

**Comment by wangzeyu135798** [comment]
In this paper, you train and finetune four models (contextpred, edgepred, infomax, making and supervised) seperately. Have you ever think about training and finetuning these five aspects simultaneously? May be the result of the model will be better ?

**Comment by mdanb** [comment]
@wangzeyu135798 although they don't test all five together, they do mention that they tried combining contextpred and masking, and that it didn't improve performance

**Comment by ha-lins** [comment]
> Hi:
> After reading your paper and code. I have two questions.
> First. In pretrain-gnns-master/model_gin, there are several supervised_.pth files, how to get them?
> Second, in README.md Fine-tuning, there is a parameter dataset, which is downstream_dataset. While in finetune.py and finetune_tune.sh , there is no this dataset parameter. Why?

Hi weihua, 

Same question! Could you pls explain what the `dataset` parameter can be? It seems not be in the finetune.py. Thanks!

**Comment by weihua916** [comment]
See [here](https://github.com/snap-stanford/pretrain-gnns/blob/master/chem/finetune.py#L107)

**Comment by weihua916** [comment]
There is no dataset argument for the bio dataset.

**Comment by zhaoshan2** [comment]
Hello, I have the same question,  could you explain how to get bio/model_gin/supervised_.pth files and how do they feed them into the future step? I didn't see them being used in fine-tuning stages. Thanks!
